README.md

```diff
@@ -36,6 +36,63 @@ Mock -> WIP -> alpha -> beta -> production
 | Native-\>Bitcoin | :x:    |
 | Bitcoin SPV      |        |
 
+## Database Interactions
```

Comment:
let's move docs to `docs/db.md` and `docs/flow.md` and here only include list of content ("index")

Reply:
Done